src/app/gantt-chart/gantt-chart.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component } from '@angular/core';
+import * as Highcharts from "highcharts/highcharts-gantt";
```

Comment:
If you are loading Highcharts Gantt / Stock / Maps like this instead of Highcharts + Gantt / Stock / Maps module then you have multiple instances of Highcharts that are not affected by the theme that is set for Highcharts and is visible only for the first demo.